package.json

```diff
@@ -23,8 +23,10 @@
     "lint:js": "eslint src/ scripts/ hak/",
     "lint:types": "tsc --noEmit",
     "build:native": "yarn run hak",
+    "build:native:universal": "yarn run hak --target x86_64-apple-darwin fetchandbuild && yarn run hak --target aarch64-apple-darwin fetchandbuild && yarn run hak --target x86_64-apple-darwin --target aarch64-apple-darwin copyandlink",
     "build32": "yarn run build:ts && yarn run build:res && electron-builder --ia32",
     "build64": "yarn run build:ts && yarn run build:res && electron-builder --x64",
+    "builduniversal": "yarn run build:ts && yarn run build:res && electron-builder --universal",
```

Comment:
I sort of wonder if this one should be `build:universal`, and then `build:32` as well... just a bit strange that some things use colon separate and others don't.